sechub-commons-model/src/test/java/com/mercedesbenz/sechub/commons/model/SecHubWebScanApiConfigurationTest.java

```diff
@@ -48,4 +52,20 @@ void json_attribute_use_is_handled_correctly_by_to_json() {
         assertEquals(expected, json);
     }
 
+    @Test
+    void api_definition_url_is_handled_correctly() throws MalformedURLException {
+        SecHubWebScanApiConfiguration config = new SecHubWebScanApiConfiguration();
+        URL apiDefinitionUrl = new URL("https://example.com/api/v1/swagger/");
+        config.setApiDefinitionUrl(apiDefinitionUrl);
+
+        /* execute */
+        String json = JSONConverter.get().toJSON(config);
+        SecHubWebScanApiConfiguration apiConfig = JSONConverter.get().fromJSON(SecHubWebScanApiConfiguration.class, json);
+
+        /* test */
+        String expected = "{\"apiDefinitionUrl\":\"https://example.com/api/v1/swagger/\",\"use\":[]}";
+        assertEquals(expected, json);
+        assertEquals(config.getApiDefinitionUrl(), apiConfig.getApiDefinitionUrl());
```

Comment:
What about a renaming here ?
```suggestion
        assertEquals(originConfig.getApiDefinitionUrl(), loadedConfig.getApiDefinitionUrl());
```